.github/workflows/ci.yml

```diff
@@ -39,14 +36,18 @@ jobs:
 
       - name: Install
         run: |
-          pip install .
+          pip install . -v
 
       - name: Test with pytest
         run: |
           pip install pytest
+          cd doc  # avoid picking up bottleneck from the source dir
           pytest --pyargs bottleneck
 
   check:
+    # This job is here is the "Required" one for merging PRs, and
+    # it only runs after all the `test` jobs above have run. Hence
+    # it serves as a check that CI actually ran before a PR gets merged.
```

Comment:
There is also another usecase: when you remove jobs (e.g. obsolete python), they stay there "dangling" and you need to remove them manually. If you make the MR depend on this job instead, then you don't need to do that anymore.